.circleci/config.yml

```diff
@@ -223,17 +223,19 @@ jobs:
       - *attach-to-workspace
       - run:
           name: Licences check
-          command: bundle exec rake ci:license_finder:run
+          command: |
+            npm dedupe
```

Comment:
Would be good to know why this need to be executed as it is not immediately obvious.

Reply:
We are installing dev dependencies in order to build the webpacker assets. But when license_finder triggers `npm list --json --long --production` it fails because some errors regarding duplicated dependencies. `npm dedupe` fixes de issue making the dependencies accessible and compatible in the environment requested

Reply:
Ok, makes sense. Could you write it down in a comment there?